httpd/handler.go

```diff
@@ -242,10 +209,12 @@ func (h *Handler) serveMetastore(w http.ResponseWriter, r *http.Request, u *User
 }
 
 // servePing returns a simple response to let the client know the server is running.
-func (h *Handler) servePing(w http.ResponseWriter, r *http.Request, u *User) {}
+func (h *Handler) servePing(w http.ResponseWriter, r *http.Request, user *influxdb.User) {
+	w.WriteHeader(http.StatusOK)
```

Comment:
Why yes it should.  204 it is.  And not minor.  I'm usually a stickler for response codes.  They really do matter.